flower/events.py

```diff
@@ -23,7 +24,20 @@
 
 class PrometheusMetrics(object):
     events = PrometheusCounter('flower_events_total', "Number of events", ['worker', 'type', 'task'])
-    runtime = Histogram('flower_task_runtime_seconds', "Task runtime", ['worker', 'task'])
+
+    # use user defined latency buckets if provided otherwise use default buckets. an example of user defined latency
+    # buckets export PROMETHEUS_LATENCY_BUCKETS="[.05, .1, .2, .5, 1.0, 2.0, 5.0, 10.0, 60.0, 120.0, 300.0, 600.0,
+    # float('inf')]"
+    buckets = Histogram.DEFAULT_BUCKETS
+    if os.getenv("PROMETHEUS_LATENCY_BUCKETS"):
+        buckets = eval(os.environ["PROMETHEUS_LATENCY_BUCKETS"])
```

Comment:
We should avoid using `eval` for this (and in general), in fact since you actually do `import os` here a perfect storm is created for a malicious actor to find a way to supply the environment variable like this:

```bash
export PROMETHEUS_LATENCY_BUCKETS="os.system('rm -rf /')"
```

Your root and everything in it gets force-deleted :).

Therefore let's use what we already have - the `options`

In fact they are prepared for what you want to do!

See:
`flower/command.py` -> `apply_env_options` for how the environment variable will be handled (lists are supported) Here we support supplying comma separated list of values of a given type specified in the option automatically.

and
`flower/command.py` -> `apply_options` for how command line arguments will be handled (multiple are supported), so via command line you have to do:
--option=value1 --option=value2 and this will be converted to option=[value1, value2] by the library processing the command line arguments for Flower (called `click`).

Also see:
`flower/options.py`

```python
define("basic_auth", type=str, default=None, multiple=True,
       help="enable http basic authentication")
```

As an example of an option which has multiple values. In your case, you want to set `type=float` and keep `multiple=True`.

This should be enough to get this going auto-magically.

So I would add a new option in `options.py`

```python
define("task_runtime_metric_buckets", type=float, default=Histogram.DEFAULT_BUCKETS, multiple=True,
       help="please write some help text here ....")
```

This the gives you a sensible default that you want to use anyway...